docs/index.bs

```diff
@@ -3145,6 +3146,22 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                   1. Let |response| be |requestResponse|'s response.
                   1. If |client| is not null, |response|'s [=response/type=] is "`opaque`", and [=cross-origin resource policy check=] with |request|'s [=request/origin=], |client|, "", and |response|'s [=filtered response/internal response=] returns <b>blocked</b>, then return null.
                   1. Return |response|.
+          1. Else if |source| is {{RouterSource/"race-network-and-fetch-handler"}}, and |request|'s [=request/method=] is \`<code>GET</code>\` then:
+
+              Note: In order to avoid duplicated fetches to |request|, the user agent may reuse |raceNetworkRequestResponse| as the result of |e|'s [=FetchEvent/potential response=], or the fallback request.
+
+              1. Run the following substeps [=in parallel=], but [=abort when=] |controller|'s [=fetch controller/state=] is "<code>terminated</code>" or "<code>aborted</code>":
+                  1. Set |raceFetchController| to the result of [=Fetch|fetching=] |request|.
```

Comment:
How |raceFetchController|'s abort is handled?

Reply:
It's just aborted, but not handled. The network request here is an additional request. `Create Fetch Event and Dispatch` has the step that informs the abort signal to the controller, which is `[=Queue a task=] to [=AbortController/signal abort=] on |controller| with |deserializedError|.`

I don't have a clear thoughts if we should send a signal to the controller like that, or it's fine to do nothing. Actually this is the same as what navigation preload does below.

Reply:
I *think* how this works is that aborting a fetch controller has two impacts:

1. Terminates the network connection. Visible to the server.
2. Communicates to any `fetch()` callers the appropriate promise rejection.

(2) is not applicable here since we are performing the fetch from the spec directly, instead of from the `fetch()` method.

So I think this is OK as written. Aborting is good because it can save server resources, but doesn't have many observable affects, and for example the error that you use doesn't matter.

Reply:
Thanks, I'll keep this as it is.